modules/custom/az_news/config/install/core.entity_view_display.node.az_news.az_feature.yml

```diff
@@ -198,7 +198,7 @@ content:
     weight: 6
     label: hidden
     settings:
-      view_mode: az_large
+      view_mode: az_card_image
```

Comment:
FYI, I'm fine with this, but be aware

**Before** 
<img width="1674" alt="image" src="https://user-images.githubusercontent.com/1023167/168182782-2977bf46-75a5-4f8e-823e-6a63c8af429d.png">

**After**
<img width="1667" alt="image" src="https://user-images.githubusercontent.com/1023167/168182653-9b79ea4a-9fc3-4aa0-a687-051e69756acd.png">

Reply:
Card image still looked good in my testing. It's also the only image style I'm aware of that doesn't show the image caption. Is there another style that doesn't show captions?

Reply:
Ugh but card also has a crop, where large does not (original intention). Do we need a new image style?

Reply:
I'm okay keeping it as a card crop for now to not introduce a new image style.